core/templates/pages/exploration-editor-page/translation-tab/services/translation-language.service.ts

```diff
@@ -64,6 +64,14 @@ export class TranslationLanguageService {
       this.activeLanguageCode);
   }
 
+  isActiveLanguageMachineTranslatable(): boolean {
+    if (!this.activeLanguageCode) {
+      return false;
+    }
+    return this.languageUtilService.languageCodeIsMachineTranslatable(
```

Comment:
It's a function, so presumably it should be changed to languageUtilService.isLanguageCodeMachineTranslatable()? (to start with a verb)